build_runner/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   build: ">=0.12.4 <0.12.7"
   build_config: ^0.3.0
   build_resolvers: ^0.2.0
-  build_runner_core: ^0.1.0
+  build_runner_core: ">=0.2.0 <0.2.1"
```

Comment:
Always good to put a comment here why this is so constrained – for other maintainers, etc

Reply:
done